README.md

```diff
@@ -115,6 +115,31 @@ The file `options.yml` contains a number of Features and Settings you can config
 - `VOICE_FEATURE` - This is an experimental feature to have spoken conversation with your assistant. It's still a bit buggy but it's coming along.
 - `GOOGLE_TOOLS_FEATURE` — This is an experimental feature that enables your assistant to access your Gmail (and soon Google Tasks and Calendar).
 
+Additionally, certain features can only be configured with environment variables. These are:
+
+- `CLOUDFLARE_STORAGE_FEATURE` - This is a feature that allows you to store message attachments in Cloudflare's R2 storage. By default this is false, but you can set it to `true` to enable it. You must configure Cloudflare first for it to work.
```

Comment:
This might be a good opportunity to explain the default, so:

```suggestion
- `CLOUDFLARE_STORAGE_FEATURE` - This is a feature that allows you to store message attachments in Cloudflare's R2 storage. When this is false, attachments are stored within postgres. This is the default configuration. But you can set it to `true` to enable it. You must configure Cloudflare Storage first for it to work.
```